src/pages/page2.jsx

```diff
@@ -9,14 +11,16 @@ class Page2 extends React.Component {
 
 	findDomNodeHandler() {
 		let bgColor = document.querySelector('.bgcolor').value;
```

Comment:
Oj nie..., podstawową zasadą w react jest to że nie manipulujesz DOMem w bezpośredni sposób ;) wszystko dzieje się przez state i react sam renderuje html. Zadziała ten kod ale jest bardzo jquerowy. 
Zamiast tego powinieneś zrobić to samo co robiliśmy dla navbara i tytułu aktualnej strony.
1. Definiujesz reducer/action dla zmiany koloru tła.
2. Wpinasz akcje zmiany koloru pod przycisk
3. W komponencie dla którego zmieniasz kolor tła czytasz ze storu wartość i ustawiasz w `style={}`